--- app/src/main/java/com/prstyadev/wibufy/ui/home/HomeScreen.kt
+++ app/src/main/java/com/prstyadev/wibufy/ui/home/HomeScreen.kt
@@ -48,9 +48,7 @@
 ) {
     val uiState by viewModel.uiState.collectAsState()
 
-    Scaffold(
-        bottomBar = {
-        }
-    ) { paddingValues ->
+    Scaffold() { paddingValues ->
         if (uiState.isLoading) {
             Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
